test/fs.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { mkdirp } = require('..');
+
+metatests.test('create 2 extra empty directories', test => {
+  mkdirp('test/ex1/ex2', '0777', err => {
+    if (err) {
+      test.bailout(err, 'Cannot create directory');
+      return;
```

Comment:
There is no need to use `return` after `test.bailout()` since the latter will throw anyway.